### ISSUE TITLE
Private mode detection using FileSystem API

### ISSUE THREAD

**Issue by jumde** [brave/brave-browser#6096]
https://bugs.chromium.org/p/chromium/issues/detail?id=990731

**Comment by tomlowenthal** [comment]
Looks like Chrome is actively fixing this, so hopefully we can wait on their fix (which won't come out before mid-March).

**Comment by fmarier** [comment]
We [disable the filesystem API](https://github.com/brave/brave-browser/issues/11407) and so this bug is no longer relevant.
